apps/schematic/api/schematic_api/controllers/storage_controller_impl.py

```diff
@@ -49,9 +64,26 @@ def get_asset_storage_class(asset_type: str) -> Callable:
     return asset_type_object
 
 
-def get_asset_view_from_schematic(
+def get_store(
     asset_type: str,  # pylint: disable=unused-argument
-) -> pd.DataFrame:
+) -> SynapseStorage:
+    """Creates a SynapseStorage class and purges its synapse cache
+
+    Args:
+        asset_type (str): The type of storage class (will be used in the future)
+
+    Returns:
+        SynapseStorage: A synapse storage class
+    """
+    access_token = get_access_token()
+    store = SynapseStorage(
+        access_token=access_token, synapse_cache_path=SYNAPSE_CACHE_PATH
+    )
+    purge_synapse_cache(store)
```

Comment:
I think I understand why I ran into an error when running endpoint: 
```
ERROR:schematic.utils.general:Cannot recognize the file size unit
INFO:werkzeug:127.0.0.1 - - [08/May/2024 18:41:44] "GET /api/v1/assetTypes/synapse/manifests/syn51078535/csv HTTP/1.1" 500 -
```
Without updating SYNAPSE_CACHE_PATH to /var/tmp/synapse, by default, the .synapseCache was saved to: /home/vscode/.synapseCache. But after the cache path update, for some reasons, when doing: 
`du -sh /var/tmp/synapse/`, it is showing 0: 

```
vscode@0b85e8ada458:/workspaces/sage-monorepo/apps/schematic/api$ du -sh /var/tmp/synapse/
0       /var/tmp/synapse/
```
I suspect similar problem will also show on AWS in deployment. It is up to you to decide if you want to have this fully figure out